### PR TITLE
Prefix and Suffix assets

### PR DESCRIPTION
**Pull request by JukeDragh**

### CODE REVIEW

File: items/suffix.txt [hunk 1]
```diff
@@ -139,3 +139,10 @@
 "Yes Man" luk=3000 offense=10 defense=10 xp=50 aReqAffirmer=1
 "World Explorer" luk=2000 xp=200 aReqExplorative=18
 "Dragon Slayer" str=5000 agi=5000 int=5000 hpregen=1250 sReqBossKills*Ukad-Aghag_the_Exile=1 sReqBossKills*White_Dragon=1 sReqBossKills*Blue_Dragon=1 sReqBossKills*Old_Copper_Dragon=1 sReqBossKills*Drunken_Dragon=1 sReqBossKills*Branor_the_Hungry=1
+"Swapped A" con=-3000 luk=-1000 crit=-100 glowing=-30 sentimentality=500 gold=20000 xp=600 salvage=5 sReqCharacter*Event*Switcheroo=500
+"Swapped B" gold=-10000 xp=-300 salvage=-2 sentimentality=500 agi=5000 con=3000 crit=100 damageReductionPercent=4 glowing=80 aegis=10
```
Comment (Kookiecraft) [inline items/suffix.txt:143]: leve-linear on damageReductionPercent = god mode in battles

Comment (seiyria) [reply items/suffix.txt:143]: thanks for the reminder. no damage reduction or damage reduction % - they're for internal use only

I've removed it from two items in here

Comment (JukeDragh) [reply items/suffix.txt:143]: Oh, it is on the no vector list here: https://github.com/IdleLands/IdleLands/blob/master/src/shared/item-generator.js#L151
so I thought it was alright.

Comment (seiyria) [reply items/suffix.txt:143]: I should add damageReduction to that. I guess it is ok in that case, I didn't realize I had it on a list... apparently I thought this through already.

Comment (JukeDragh) [reply items/suffix.txt:143]: I think Kookie added it after they did Req stuff

**Pull request merged**